app/javascript/home/Home.vue

```diff
@@ -64,7 +64,7 @@ function setScrollToFragmentTimeouts() {
 </script>
 
 <style lang="scss">
-@import 'css/variables';
+@use 'css/variables' as *;
```

Comment:
This change wasn't made automatically (which makes sense, because I guess these files aren't "downstream" from the `app/assets/stylesheets/styles.scss` entrypoint file that I provided to the migrator tool).